lib/vertex_client/connection.rb

```diff
@@ -30,6 +30,7 @@ def transform_payload
     # TODO Consider removing this conditional to make this more robust
     def handle_response(response)
       if response
+        return TaxAreaResponse.new(response, @payload.response_key) if @payload.is_a? TaxAreaPayload
```

Comment:
Thinking that long term we may want to have Payloads able to handle their own responses as a way to clean up this nasty bit.